Service/Form/Submission/FormSubmissionRequest.php

```diff
@@ -22,7 +22,7 @@ final class FormSubmissionRequest
     private $files;
     /** @var string */
     private $label;
-    /** @var string */
+    /** @var \DateTime|false */
```

Comment:
false or null?

Reply:
I had null at first but Travis wouldn't allow it.